emitter.py

```diff
@@ -29,7 +30,13 @@
 
 
 def remove_control_chars(s):
-    return control_char_re.sub('', s)
+    if isinstance(s, str):
+        sanitized = control_char_re.sub('', s)
+    elif isinstance(s, unicode):
+        sanitized = ''.join(['' if unicodedata.category(c) == 'Cc' else c
```

Comment:
I'm not sure if it's the only category that might cause `json.dumps()` to fail, I do know it's the equivalent of the ASCII portion of remove_control_chars()... I could also remove `Cf`, which are likely also dangerous. I do that in the `github` and `bitbucket` integrations.